x-pack/plugins/observability_solution/slo/server/lib/rules/slo_burn_rate/register.ts

```diff
@@ -100,6 +100,10 @@ export function sloBurnRateRuleType(
         { name: 'sloName', description: sloNameActionVariableDescription },
         { name: 'sloInstanceId', description: sloInstanceIdActionVariableDescription },
         { name: 'suppressedAction', description: suppressedActionVariableDescription },
+        {
+          name: 'errorBudgetExhaustionTime',
```

Comment:
```suggestion
          name: 'errorBudgetExhaustionTimeInHours',
```
When I used the variable, I just got `7.2` so I think it's helpful to let the user know the units. 

Or, instead of returning a simple number you could return the value with the units.

Reply:
I wonder what the other rules are doing, variable name or { value, unit } ?
When I think about the use case, I think it's easier for customers to write conditions with a variable using always the same unit, than an object {value,unit}. But personnaly prefer the later as a model